src/ONYXKEYS.js

```diff
@@ -45,6 +45,9 @@ export default {
     // Contains all the personalDetails the user has access to
     PERSONAL_DETAILS: 'personalDetails',
 
+    // Contains all the personalDetails the user has access to, keyed by accountID
+    PERSONAL_DETAILS_LIST: 'personalDetailsList',
```

Comment:
I'm confused why this is showing in the diff since this key also  exists on `main`. Maybe a GitHub bug? We should merge main into this branch to be safe